bin/setup

```diff
@@ -20,9 +20,6 @@ FileUtils.chdir APP_ROOT do
   # Install JavaScript dependencies if using Yarn
   # system('bin/yarn')
 
-  puts "\n== Copying sample files =="
-  cp_r 'config/examples/.', 'config/', preserve: true, dereference_root: true
```

Comment:
I think we should not overwrite files, is there such an option?

Reply:
nothing, mistake

Reply:
Well, after https://github.com/3scale/porta/pull/3766/commits/b9dd0224590666c08d39ac7fe3ad6b124bb5a446 this change has been reverted so we are overwriting again @akostadinov